pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetStudiesRoute.java

```diff
@@ -40,14 +40,14 @@
 
     @Override
     public Object handle(Request request, Response response) throws Exception {
-        final String umbrellaIdentifier = request.queryParams(RouteConstants.QueryParam.UMBRELLA);
+        String umbrellaGuid = request.queryParams(RouteConstants.QueryParam.UMBRELLA_GUID);
```

Comment:
We can leave the old query parameter name (`name`) and prevent breaking the front-end. However, `name` containing a GUID sounds wrong. What's more important here - backward compatibility or correctness?